MaterialComponentsSnapshotTests.podspec

```diff
@@ -45,7 +45,7 @@ module SnapshotPodspecHelper
 
   def self.components
     return Dir["components/**/tests/snapshot"].map { |dir|
-      dir = Component.new(dir.split(File::SEPARATOR)[-3])
+      dir = Component.new(dir.split(File::SEPARATOR)[1])
```

Comment:
@wenyuzhang666  How does this work for schemes? Why not use -3?

Reply:
It doesn't work for schemes though scheme currently doesn't have snapshot tests. 1 is used to directly extract the directory name under `components`. Both index 1 and -3 are not good assumptions.